helm/net-exporter-chart/values.yaml

```
@@ -4,6 +4,9 @@ namespace: monitoring
 userID: 1000
 groupID: 1000
 
+port: 8000
+giantswarmcidr: 137.117.188.104/32
```

```suggestion
giantswarmCIDR: 137.117.188.104/32
```

What is this network range? AIUI we can have comments in the values file and I think that is needed here. I don't follow what this is for.

---

Added comment regarding that value